tests/testAutopas/tests/autopasInterface/Newton3OnOffTest.cpp

```diff
@@ -32,7 +32,7 @@ INSTANTIATE_TEST_SUITE_P(
 
               // container factory
               autopas::ContainerSelector<Particle, FPCell> containerSelector({0., 0., 0.}, {10., 10., 10.}, 1.);
-              autopas::ContainerSelectorInfo containerInfo(1., 0.);
+              autopas::ContainerSelectorInfo containerInfo(1., 0., 64);
```

Comment:
is 64 really what we want here?

Reply:
it's sth. that works with cuda, so yes !

Reply:
and it was the previous default value.